wiki/Beatmaps/Packs/en.md

```diff
@@ -0,0 +1,48 @@
+# Beatmaps Packs
+
+_Main page: [Beatmaps](/wiki/Beatmaps)_
+
+Beatmaps packs, as the title suggests, are a `.zip` files containing beatmaps.
+The name of the pack is based on what they contain (e.g. `Approved Beatmap Pack #7` would only contain [approved](#approved) beatmaps).
+It is worth noting that the most packs are:
+
+- Roughly either ~100MB (Taiko/osu!mania) or ~200MB (-/Approved)
+- Always contain videos (if included)
+- The beatmaps are either in the form of `.zip` or `.osz` file
+
+## Categories
+
+Beatmaps packs split into 4 categories:
+
+| Category | Beatmap types |
+| -------- | ------------- |
+| Standard | Ranked or Approved songs |
+| Chart | Fortnightly Chart challenge songs |
+| Theme | General theme of the songs |
+| Artist/Album | Based on Artist/Album |
+
+## Obtaining Every Beatmap
+
+This is for those who really want to *fully complete your song list* (and not just download your favorite songs).
+
+Keep in mind that you will need quite a lot of digital space to storage these beatmaps.
+As beatmap packs (and torrents) also downloads with the video and other beatmap skinable elements.
+
+### Beatmap Pack Listing
+
+Download all the (-/Approved/Taiko/osu!mania) beatmaps packs from the **Standard** category and nowhere else.
```

Comment:
once again, that's the pack name and not fully referring to the game mode itself